Sources/HaCWebsite/main.swift

```diff
@@ -1,3 +1,15 @@
 import HaCWebsiteLib
+import DotEnv
+import Foundation
 
-serveWebsite()
+DotEnv.load()
+
+// Check for required env vars
+guard let _ = DotEnv.get("DATA_DIR") else {
+  fatalError("Missing environment variable")
```

Comment:
Should we colocate these checks with the code that uses the variable? e.g. in `WorkshopManager`. We could add a `requireEnvVar` to our [`Config`](https://github.com/hackersatcambridge/hac-website/blob/master/Sources/HaCWebsiteLib/configuration.swift) struct as well.